truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity 0.5.10;
+
+import './Stoppable.sol';
+
+contract Splitter is Stoppable {
+    
+    struct Account {
+        uint    pendingWithdrawals;
+    }
+    
+    mapping(address => Account) accountStorage;
```

Comment:
I'm sure you noticed the `struct` only has one element in it now, so `mapping(address => uint)...` would carry the day.